robusta_krr/main.py

```diff
@@ -282,7 +282,7 @@ def run_strategy(
                         kind=inspect.Parameter.KEYWORD_ONLY,
                         default=typer.Option(
                             field_meta.default,
-                            f"--{field_name}",
+                            f"--{field_meta.field_info.extra.get('param_decls') or field_name}",
```

Comment:
Is `param_decls` some documented thing in pydantic? Maybe better to use `alias`?

Reply:
it won't work, I checked